src/global_styling/mixins/_responsive.test.ts

```diff
@@ -63,130 +53,47 @@ describe('useEuiBreakpoint', () => {
-        expect(
-          // @ts-expect-error deliberate incorrect type
-          testCustomHook(() => useEuiBreakpoint(['teeny-tiny', 'SUPERMASSIVE']))
-            .return
-        ).toEqual(fallbackOutput);
-      });
-
-      it('warns when invalid size numbers are passed', () => {
-        expect(
-          testCustomHook(() => useEuiBreakpoint([100, 400])).return // we might support this someday, but today is not that day
-        ).toEqual(fallbackOutput);
-      });
-
-      it('warns when 0 and Infinity are used in the wrong position', () => {
-        expect(
-          testCustomHook(() => useEuiBreakpoint([Infinity, 0])).return
-        ).toEqual(fallbackOutput);
-      });
-    });
+  it('does not generate a min-width if the min size is xs', () => {
+    expect(
+      testCustomHook(() => useEuiBreakpoint(['xs', 'm'])).return
+    ).toMatchInlineSnapshot('"@media only screen and (max-width: 991px)"');
+  });
 
-    describe('invalid breakpoint size order', () => {
-      afterEach(() => {
-        expect(consoleStub).toHaveBeenCalledTimes(1);
-        expect(consoleStub).toHaveBeenLastCalledWith(
-          'Invalid breakpoint sizes passed. The first size should be smaller than the last size'
-        );
-      });
+  it('skips generating a max-width if the max size is xl', () => {
+    expect(
+      testCustomHook(() => useEuiBreakpoint(['m', 'xl'])).return
+    ).toMatchInlineSnapshot('"@media only screen and (min-width: 768px)"');
+  });
 
-      it('warns if min breakpoint is not smaller than the max breakpoint', () => {
-        expect(
-          testCustomHook(() => useEuiBreakpoint(['xl', 'xs'])).return
-        ).toEqual(fallbackOutput);
-      });
+  describe('fallback behavior', () => {
+    const fallbackOutput = '@media only screen';
 
-      it('warns if the min/max breakpoints are equal', () => {
-        expect(
-          testCustomHook(() => useEuiBreakpoint(['s', 's'])).return
-        ).toEqual(fallbackOutput);
-      });
+    test('if at least one input is not passed', () => {
+      // @ts-expect-error Source has 0 element(s) but target requires 1
+      expect(testCustomHook(() => useEuiBreakpoint([])).return).toEqual(
+        fallbackOutput
+      );
     });
 
-    describe('valid but funky breakpoint combos', () => {
-      afterEach(() => {
-        expect(consoleStub).not.toHaveBeenCalled();
-      });
-
-      test('0 and xs', () => {
-        // Since xs is (currently) already 0, this should output nothing
-        expect(
-          testCustomHook(() => useEuiBreakpoint([0, 'xs'])).return
-        ).toEqual(fallbackOutput);
-      });
-
-      test('xs and infinity', () => {
-        // Since xs is (currently) 0, there should be no min or max width
-        expect(
-          testCustomHook(() => useEuiBreakpoint(['xs', Infinity])).return
-        ).toEqual(fallbackOutput);
-      });
+    test('if at least one input is not passed', () => {
```

Comment:
woops, duplicate test name
```suggestion
    test('if an invalid breakpoint size is passed', () => {
```